.hlint.yaml

```
@@ -1,4 +1,18 @@
 - arguments: [-XTypeApplications, -XNumericUnderscores]
 # Often, code is more readable with extra parens, or less readable with list comprehensions
 - ignore: {name: "Redundant bracket"}
-- ignore: {name: "Use list comprehension"}
\ No newline at end of file
+- ignore: {name: "Use list comprehension"}
+
+# Import Preferences
+- modules:
+  - {name: Data.Set, as: Set, message: "Use complete name for qualified import of Set"}
+  - {name: Data.Map, as: Map, message: "Use complete name for qualified import of Map"}
+  - {name: Data.Text, as: Text, message: "Use complete name for qualified import of Text"}
+
+# Styling Preferences
+- suggest: {lhs: "asum [x, y]", rhs: x <|> y}
+- suggest: {lhs: return x, rhs: pure x}
+- suggest: {lhs: () <$ x, rhs: Control.Monad.void x}
+- suggest: {lhs: x <&> f, rhs: f <$> x}
+- suggest: {lhs: Data.Text.pack x, rhs: Data.String.Conversion.toText x, note: "For consistency use, Data.String.Conversion module instead!"}
+- suggest: {lhs: Data.Text.unpack x, rhs: Data.String.Conversion.toString x, note: "For consistency use, Data.String.Conversion module instead!"}
```

Don't justify your choices in the warnings.  Justifications should be in a style guide, and they can be left out until a style guide is ready, as long as we **internally** agree on correct decisions.
```suggestion
- suggest: {lhs: Data.Text.pack x, rhs: Data.String.Conversion.toText x}
- suggest: {lhs: Data.Text.unpack x, rhs: Data.String.Conversion.toString x}
```